ssc/cardano-sl-ssc.cabal

```diff
@@ -55,6 +55,7 @@ library
                      , mtl
                      , node-sketch
                      , parsec
+                     , rocksdb-haskell >= 1.0.1
```

Comment:
Why did it become necessary? You didn't change anything in `ssc/`, so it looks strange.

Reply:
Hmm, wouldn't it cause problems to depend on a version of a package in which a problem exists, and for which a solution has been found and made available?

If we bump in one place, wouldn't it make sense to bump elsewhere as well?

Reply:
But the thing is: `cardano-sl-ssc` didn't depend on `rocksdb-haskell` before your changes.

Reply:
Oh, wait - it does, but in `master`. My bad. I `cherry-picked` this command from `master` to this branch after I closed the previous PR. Let me fix this.